File: backend/models/organization_model.py
from enums.app_enum import StatusEnum
from .model_base import BaseModel


class OrganizationModel(BaseModel):

    id = 0
    name = None
    contact = None
    tel = None
    status = None
    create_time = None

    def __init__(self, **kw):
        self.id = kw.get('id')
        self.name = kw.get('name')
        self.contact = kw.get('contact')
        self.tel = kw.get('tel')
        self.status = kw.get('status')
        super().__init__()

File: frontend/or/src/app/shared/models/src/organization.model.ts
import { StatusEnum } from "@app/shared/enums/app.enum";
import { BaseModel, BaseSearchModel } from "./base.model";

export class Organization extends BaseModel {
  id?: number;
  name?: string;
  contact?: string;
  tel?: string;
  status: StatusEnum;
  statusName?: string;
  enabled?: boolean;
}

export class OrganizationSearch extends BaseSearchModel {
}
